daprdocs/content/en/operations/observability/metrics/metrics-overview.md

```diff
@@ -84,9 +92,10 @@ kind: Configuration
 metadata:
   name: daprConfig
 spec:
-  metric:
-      enabled: true
-      rules:
+  metrics:
+    enabled: true
+    increasedCardinality: true
+    rules:
       - name: dapr_runtime_service_invocation_req_sent_total
```

Comment:
- Does dapr_runtime_service_invocation_req_sent_total this need to match a metric name? Presumably not.

Reply:
Yes that's the metric name. However, here too, this is something we shouldn't be promoting much as it's a legacy thing.

Reply:
But does it need to name match with the metric name or not?

Reply:
I *think* so, according to the docs. I'm not super familiar with that feature myself

Reply:
Can I suggest then that we add this, which if used, is a good practice anyway. That the name needs to match the metrics name explicitly.